test/integration/serverTest.js

```diff
@@ -225,6 +225,26 @@ describe('Brave Sensor server', () => {
       })
     })
 
+    it('should send a 200 response if a sirenAddressed call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      const response = await chai.request(server).post('/api/sirenAddressed').send({})
+      expect(response).to.have.status(200)
+    })
+
+    it('should send a 200 response if a sirenEscalated call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      const response = await chai.request(server).post('/api/sirenEscalated').send({})
+      expect(response).to.have.status(200)
+    })
+
+    it('should start an alert session if a sirenEscalated call is receieved', async () => {
```

Comment:
is this actually the case? As I understand it, we want to start an alert session when the Sensor raises a DURATION or STILLNESS alert (which happens in the sensorEvent endpoint). When the sirenEscalated endpoint is hit, my understanding is that we update an existing session, and escalate to the fallback number"? Does that make sense?

Reply:
I had originally suggested to Maria that starting the regular chatbot session might be a good way to escalate the Siren, but I have changed my mind and now thing that just hitting the fallback would be better. We were waiting to hear your thoughts on it, but it sounds like you're also thinking that a fallback at this point would be the way to go, is that right?

Reply:
Yes, just looked into it and you're right. A session is started as soon as the siren is called.